packages/bpk-component-boilerplate/src/__snapshots__/BpkBoilerplate-test.tsx.snap

```diff
@@ -1,6 +1,6 @@
 // Jest Snapshot v1, https://goo.gl/fbAQLP
 
-exports[`BpkBoilerplate should render correctly 1`] = `
+exports[`BpkBoilerplate should not support custom class names 1`] = `
```

Comment:
what part of this test asserts we don't support className?

Reply:
Currently its just this snapshot, as that is how most of our tests are setup for in Backpack, something that as a future initative we need to invest the time into doing more assertions for tests.

However here this is just the boilerplate so it's not necessarily key as all the tests here and written in the test file are subject to change when someone generates a new component and modifies towards the component they build

Reply:
@runmoore have made some improvements to the tests now to not rely on snapshot testing for such a check :)